changelogs/unreleased/add-alpine-3.16-ci.md

```diff
@@ -1,3 +1,3 @@
 ## feature/build
 
-* Support Alpine 3.16 build.
+* Alpine Linux 3.16 is now supported.
```

Comment:
I'm not sure that we should declare that we support Alpine. In fact, the build is added into CI, but the tests were not enabled and there are some failed ones. We didn't analize them ATM. @ylobankov may know more and may suggest a proper wording, I think.